source/mainnet/net/guides/run-node-macos.rst

```diff
@@ -286,6 +286,9 @@ Since the GRPC V2 port is different than the GRPC V1 port, you need to change it
 
 .. _configure-baker-macos:
 
+.. Note::
+   If the node is configured with TLS, then `CONCORDIUM_NODE_COLLECTOR_GRPC_HOST` must be configured such that it uses the domain of the certificate.
```

Comment:
It would probably make sense to link to the instructions on how to configure TLS, here https://github.com/Concordium/concordium-node/blob/main/docs/grpc2.md#grpc-api-v2

Reply:
And in all the instances below as well.